processor.go

```diff
@@ -186,8 +186,18 @@ func (p *Processor) process(ctx gocontext.Context, buildJob Job) {
 	ctx, span := trace.StartSpan(ctx, "ProcessorRun")
 	defer span.End()
 
+	job_id := buildJob.Payload().Job.ID
+	repo := buildJob.Payload().Repository.Slug
+	infra := p.config.ProviderName
+	site := p.config.TravisSite
+
 	span.AddAttributes(
+
 		trace.StringAttribute("app", "worker"),
+		trace.Int64Attribute("JobID", (int64(job_id))),
+		trace.StringAttribute("repo", repo),
```

Comment:
these could probably be simplified a bit by inlining the variables from above, wdyt?

Reply:
e.g.

```
trace.StringAttribute("repo", buildJob.Payload().Repository.Slug),
```